qa/integration-tests/src/test/java/io/camunda/zeebe/it/management/ExportingEndpointIT.java

```diff
@@ -0,0 +1,171 @@
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.it.management;
+
+import static io.camunda.zeebe.it.management.ExportingEndpointIT.StableValuePredicate.hasStableValue;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+import feign.FeignException;
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.model.bpmn.Bpmn;
+import io.camunda.zeebe.qa.util.actuator.ExportingActuator;
+import io.camunda.zeebe.qa.util.actuator.PartitionsActuator;
+import io.camunda.zeebe.qa.util.testcontainers.ZeebeTestContainerDefaults;
+import io.zeebe.containers.cluster.ZeebeCluster;
+import io.zeebe.containers.exporter.DebugReceiver;
+import java.time.Duration;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.Predicate;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.Network;
+
+final class ExportingEndpointIT {
+
+  @Test
+  void shouldPauseExporting() {
+    final var exportedRecords = new CopyOnWriteArrayList<>();
+    try (final var receiver = new DebugReceiver(exportedRecords::add).start()) {
+      try (final var cluster =
+          ZeebeCluster.builder()
+              .withImage(ZeebeTestContainerDefaults.defaultTestImage())
+              .withEmbeddedGateway(true)
+              .withBrokerConfig(
+                  zeebeBrokerNode ->
+                      zeebeBrokerNode.withDebugExporter(receiver.serverAddress().getPort()))
+              .withBrokersCount(3)
+              .withPartitionsCount(3)
+              .withReplicationFactor(3)
+              .build()) {
+        cluster.start();
+
+        try (final var client = cluster.newClientBuilder().build()) {
+          deployProcess(client);
+          startProcess(client);
+
+          final var recordsBeforePause =
+              Awaitility.await()
+                  .atMost(Duration.ofSeconds(30))
+                  .during(Duration.ofSeconds(5))
+                  .until(exportedRecords::size, hasStableValue());
+
+          // when
+          ExportingActuator.of(cluster.getAvailableGateway()).pause();
+          startProcess(client);
+
+          // then
+          Awaitility.await()
+              .atMost(Duration.ofSeconds(30))
+              .during(Duration.ofSeconds(10))
+              .failFast(() -> assertThat(exportedRecords).hasSize(recordsBeforePause));
+
+          Awaitility.await().untilAsserted(() -> allPartitionsPaused(cluster));
+        }
+      }
+    }
+  }
+
+  @Test
+  void failsIfMemberIsShutdown() {
```

Comment:
:x: This can  be removed. Failure scenarios are handled in `ExportingControlServiceTest`